phpunit/functional/DocumentTest.php

```diff
@@ -900,22 +942,22 @@ public function testGetDuplicateOf()
         ];
         $document = new \Document();
         $document->add($input);
-        $this->boolean($document->isnewItem())->isFalse();
+        $this->assertFalse($document->isnewItem());
 
-       // Check the file is found in the FB
-        $instance = $this->newTestedInstance();
-        $output = $instance->getDuplicateOf(0, __DIR__ . '/../fixtures/uploads/foo.png');
-        $this->boolean($output)->isTrue();
+        // Check the file is found in the FB
+        $instance = new \Document();
+        $output = $instance->getDuplicateOf(0, __DIR__ . '/../../tests/fixtures/uploads/foo.png');
+        $this->assertTrue($output);
 
-       // togle the blackisted flag
+        // toggle the blacklisted flag
         $success = $instance->update([
             'id'             => $instance->getID(),
             'is_blacklisted' => '1'
         ]);
-        $this->boolean($success)->isTrue();
+        $this->assertTrue($success);
 
-       // Test when the document exists and is blacklisted
-        $output = $instance->getDuplicateOf(0, __DIR__ . '/../fixtures/uploads/foo.png');
-        $this->boolean($output)->isFalse();
+        // Test when the document exists and is blacklisted
+        $output = $instance->getDuplicateOf(0, __DIR__ . '/../../tests/fixtures/uploads/foo.png');
```

Comment:
Same (I may have missed more, maybe you can check with a search for ../tests/fixtures ?)